.shellcheckrc

```diff
@@ -0,0 +1 @@
+disable=SC1090,SC1091 # Ignore warnings about being unable to follow sourced files
```

Comment:
When you set 
```suggestion
external-sources=true
```

here, `shellcheck` should follow external sources (might need `# shellcheck source=somefile` in some places).
See the 'Rationale' section here: https://www.shellcheck.net/wiki/SC1091

Reply:
I'm fine with ignoring these two warnings entirely - we always have done, this just makes it a more global change without having to put the directive in any file that needs it.

None of the paths it complains about are relative to the repo from a development point of view - applying the `source=` directive will not help in any of these cases

Reply:
Ok, let's go on with this, I might need to tweak in a follow-up PR